content/200-concepts/150-data-platform/08-billing/05-invoices.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: 'Invoices'
+metaTitle: 'Invoices'
+metaDescription: 'The Prisma Data Platform uses Stripe as the payments provider and it is Stripe that issues and sends the monthly invoices when you have a paid plan for a project.'
+---
+
+<TopBlock>
+
+The Prisma Data Platform uses Stripe as the payments provider and it is Stripe that issues and sends the monthly invoices when you have a paid plan for a project.
```

Comment:
I believe we should link to stripe.com even it may be obvious which Stripe we're referring to.

Reply:
Fixed.